handler/api/api.go

```diff
@@ -345,6 +345,7 @@ func (s Server) Handler() http.Handler {
 	r.Route("/builds", func(r chi.Router) {
 		r.Use(acl.AuthorizeAdmin)
 		r.Get("/incomplete", globalbuilds.HandleIncomplete(s.Repos))
+		r.Get("/running_status", globalbuilds.HandleRunningStatus(s.Repos))
```

Comment:
maybe `/incomplete/v2` for now, and then we rename to `/incomplete` and deprecated the old endpoint in a future release?